server/eof/api.ts

```diff
@@ -1,11 +1,12 @@
 import * as R from 'ramda'
 
 import { ApiAuthMiddleware } from '@server/api/middleware'
+import { readEofOdps } from '@server/repository/odp/readEofOdps'
+import { readFocOdps } from '@server/repository/odp/readFocOdps'
```

Comment:
why not to use OdpService?

Reply:
This file needs refactoring anyway (still part of apiRouter), fixing for now